src/components/select/select-screen.js

```diff
@@ -76,6 +76,7 @@ export class Select extends Screen {
 
             config.suffix && (button.config.ariaLabel = [button.config.ariaLabel, config.suffix].join(" "));
             config.enabled === false && button.off(Phaser.Input.Events.POINTER_UP);
+            config.enabled === false && button.off(Phaser.Input.Events.POINTER_OVER);
```

Comment:
haha is it called enabled in select screen and disabled in shop? 😂

Reply:
yes